src/item_collaborative_filtering.py

```diff
@@ -23,53 +23,98 @@ def __generate_item_pairs(self, df, item):
         :return: list of tuples, length=df['item'].nunique()
         """
         if item is not None:
-            return [(item, paired_item) for paired_item in df[self.item_column].unique() if paired_item!=item]
+            return [(item, paired_item) for paired_item in df[self.item_column].unique() if paired_item != item]
         else:
             return [(item, paired_item) for
                     item, paired_item in permutations(df[self.item_column].unique(), 2)]
 
+    def __item_users(self, df):
```

Comment:
Since this is a mutable function (it modifies the instance data instead of just returning something), it'd be good to name it as an action, e.g. `calculate_item_users` or `initialize_item_users_map`